internal/store/store_test.go

```diff
@@ -20,10 +28,119 @@ type storeTest struct {
 	myBP             blueprint.Blueprint
 	CommitHash       string
 	myChange         blueprint.Change
+	myManifest       distro.Manifest
+	myPipeline       osbuild.Pipeline
+	myPipeline2      osbuild.Pipeline
+	myBuild          osbuild.Build
+	myTarget         *target.Target
+	mySources        map[string]osbuild.Source
+	ImageType        myImageType
+	myCompose        Compose
+	myImageBuild     ImageBuild
+	mySourceConfig   SourceConfig
+}
+type myImageType struct {
+	Arches *myArch
+}
+type myArch struct {
+	Distros    *myDistro
+	ImageTypes *myImageType
+}
+type myDistro struct {
+	Arches *myArch
+}
```

Comment:
Shouldn't these data types be shared between test & SUT or am I not seeing the bigger picture here ?

Reply:
@teg actually told me that all of this is already defined under `distro/test_distro` so I'm going to change it once again. Will make it look much simpler. I made this change in a separate branch so I just need to figure out how to get it to this branch and push it.